roles/redpanda_broker/templates/configs/defaults.j2

```diff
@@ -74,6 +74,14 @@
       "tune_ballast_file": true
     },
     "pandaproxy": {},
-    "schema_registry": {}
+    "schema_registry": {
+        "schema_registry_api": [
+        {
+          "address": "{{ hostvars[inventory_hostname].private_ip }}",
```

Comment:
does this need to be accessible externally like kafka_api or is this something that generally needs to remain private?

Reply:
Needs to be accessible externally, I changed to advertised_ip. Thanks